.github/workflows/release.yaml

```
@@ -403,7 +403,7 @@ jobs:
           token: ${{ secrets.CDRCI_GITHUB_TOKEN }}
           repository: coder/packages
           event-type: coder-release
-          client-payload: '{"coder_version": "${{ steps.version.outputs.version }}"}'
+          client-payload: '{"coder_version": "${{ steps.version.outputs.version }}", "is_stable": "${{ $CODER_RELEASE_CHANNEL == "stable" }}"}'
```

```suggestion
          client-payload: '{"coder_version": "${{ steps.version.outputs.version }}", "is_stable": "${{ inputs.release_channel == "stable" }}"}'
```
Not sure if this is syntactically correct, but I think this should use the input vs shell env.